wurst/objects/items/Items.wurst

```diff
@@ -122,4 +77,28 @@ public class ItemChannelCast extends ChannelAbilityPreset
         this.setManaCost(1, 0)
         this.setCooldown(1, cooldown)
         this.setCastRange(1, castRange)
+        this.setTargetsAllowed(1, targetsAllowed)
         this.setEditorSuffix("(Wurst)")
+
+/**
+    Use this to create an item spell who doesn't play an animation
+    Contrary to the Channel ability, WindWalk do not stop the unit current order when casted.
+**/
+public function createItemWindWalkCast(int newId) returns AbilityDefinitionWindWalk
+    return new AbilityDefinitionWindWalk(newId)
+        ..setArtCaster("")
+        ..setArtEffect("")
+        ..setArtTarget("")
+        ..setArtSpecial("")
+        ..setHeroAbility(false)
+        ..setItemAbility(true)
+        ..setLevels(1)
+        ..setTooltipNormal(1, "dummy spell")
+        ..setTooltipNormalExtended(1, "dummy spell")
+        ..setManaCost(1, 0)
+        ..setAnimationNames("")
+        ..setDurationHero(1, 0.01)
+        ..setCastingTime(1, 0)
+        ..setDurationNormal(1, 0.01)
```

Comment:
Does this not work at 0? And can you set the fade time higher than this if not? I worry that fully fading for any time at all will break attack / patrol orders on that unit for the enemy.

Reply:
Unit goes invisible if it's set at 0.